.docker/base-with-services/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# syntax=docker/dockerfile:1
+FROM base
+
+LABEL maintainer="AiiDAlab Team <aiidalab@materialscloud.org>"
+
+USER root
+WORKDIR /opt/
+
+ARG PGSQL_VERSION
+ARG RMQ_VERSION
+
+ENV PGSQL_VERSION=${PGSQL_VERSION}
+ENV RMQ_VERSION=${RMQ_VERSION}
+
+RUN mamba install --yes \
+     --channel conda-forge \
+     postgresql=${PGSQL_VERSION} && \
+     mamba clean --all -f -y && \
+     fix-permissions "${CONDA_DIR}" && \
+     fix-permissions "/home/${SYSTEM_USER}"
+
+# Install erlang.
+RUN apt-get update --yes && \
+     apt-get install --yes --no-install-recommends \
+     erlang \
```

Comment:
It is tricky to install the correct RMQ version because the erlang version needs to compatible with the final rabbitmq version as shown in https://www.rabbitmq.com/which-erlang.html